scss/upload/_layout.scss

```diff
@@ -1 +1,6 @@
-@import "~@progress/kendo-theme-default/scss/upload/layout";
+$dropzone-upload-status-icon-margin: $padding-y !default;
+$upload-status-margin: 0 !default;
+$file-name-margin: 0 !default;
+$invalid-icon-margin: $padding-x-sm / 2 !default;
```

Comment:
If we are using `$*-icon-offset`, shouldn't we change the `$icon-spacing` in the global variables files?